src/routes/plugin-global-rules/detail.$id.tsx

```diff
@@ -0,0 +1,9 @@
+import { createFileRoute } from '@tanstack/react-router'
+
+export const Route = createFileRoute('/plugin-global-rules/detail/$id')({
+  component: RouteComponent,
+})
+
+function RouteComponent() {
+  return <div>Hello "/plugin-global-rules/detail/$id"!</div>
```

Comment:
Why does it use `plugin-global-rules` instead of `global-rules` (at least for SPA routing)?